%YAML:1.0
---

# 哨兵下云台
KE0200110076:
    #内参矩阵
    Intrinsic: [1557.387791, 0.000000, 646.397848, 
                0.000000, 1559.413404, 522.577804, 
                0.000000, 0.000000, 1.000000]
    #畸变参数矩阵，k1,k2,k3,p1,p2
    Coeff: [-0.094455, 0.257714, 0.001787, 0.001353, 0.000000]
    #x,y,z坐标偏移量(相机至枪管),单位m
    xyz_offset: [0,0,0]
    #yaw,pitch角度偏移量,单位度
    angle_offset: [0.5,1]
    #使用迭代法求解pitch补偿的最大迭代次数
    max_iter: 10
    #停止迭代的最小误差(单位m)
    stop_error: 0.001
    #龙格库塔法求解落点的迭代次数
    R_K_iter: 60

    #陀螺仪到相机的变换矩阵 R_c = T_ci * R_i
    T_ci: [0.0106869, 0.99950186, -0.02969544, 0,
        0.01352137, 0.02954998, 0.99947185, 0.1458,
        0.99985147, -0.01108278, -0.01319884, -0.13,
        0,              0,              0,              1]
    #相机到陀螺仪的变换矩阵 R_i = T_ic * R_c
    T_ic: [ 0.0106869, 0.01352137, 0.99985147, 0.13,
        0.99950186, 0.02954998, -0.01108278, -0,
        -0.02969544, 0.99947185, -0.01319884, -0.1458,
        0,            0,              0,              1]
    #陀螺仪至云台转轴中心平移向量 R_g = R_i - T_iw
    T_iw: [0.02549, 0, -0.09566]

# 步兵
KE0200110073:
    #内参矩阵
    Intrinsic: [1269.563633388266,  0.000000000000,  661.453127484648,
                    0.000000000000,  1271.597581984607,  532.581375064043,
                    0.000000000000,  0.000000000000,  1.000000000000]
    #畸变参数矩阵，k1,k2,k3,p1,p2
    Coeff: [-0.18667314595589776, 0.12988512986235917, 0, 0.00570949961988075, -0.0008511964245533121]
    #x,y,z坐标偏移量(相机至枪管),单位m
    xyz_offset: [0,0,0]
    #pitch,yaw角度偏移量,单位度
    angle_offset: [0,0]
    #使用迭代法求解pitch补偿的最大迭代次数
    max_iter: 10
    #停止迭代的最小误差(单位m)
    stop_error: 0.001
    #龙格库塔法求解落点的迭代次数
    R_K_iter: 50
    #陀螺仪到相机的变换矩阵 R_c = T_ci * R_i
    T_ci: [1, 0, 0, 0,
            0, 1, 0, 0,
            0, 0, 1, 0,
            0, 0, 0, 1]
    #相机到陀螺仪的变换矩阵 R_i = T_ic * R_c
    T_ic: [1, 0, 0, 0,
            0, 1, 0, 0,
            0, 0, 1, 0,
            0, 0, 0, 1]
    #陀螺仪至云台转轴中心平移向量 R_w = R_i - T_iw
    T_iw: [0, 0, 0]

# 英雄
KE0200110075:
    #内参矩阵
    Intrinsic: [1280.220400,  0.000000000000,  690.800519,
                0.000000000000,  1275.152048,  493.367743,
                0.000000000000,  0.000000000000,  1.000000000000]
    Coeff: [-0.204890, 0.155752, 0.002706, 0.006799, 0] 
    #x,y,z坐标偏移量(相机至枪管),单位m
    xyz_offset: [0,0,0]
    #pitch,yaw角度偏移量,单位度
    angle_offset: [2,1]
    #使用迭代法求解pitch补偿的最大迭代次数
    max_iter: 10
    #停止迭代的最小误差(单位m)
    stop_error: 0.001
    #龙格库塔法求解落点的迭代次数
    R_K_iter: 50
    #陀螺仪到相机的变换矩阵 R_c = T_ci * R_i
    T_ci: [-0.00907793, -0.99988789, -0.01190754, 0.01763271,
            -0.02731819, 0.01215157, -0.99955293, 0.08052791,
            0.99958557, -0.00874858, -0.02742544, -0.25691905,
            0, 0, 0, 1]
    #相机到陀螺仪的变换矩阵 R_i = T_ic * R_c
    T_ic: [-0.00907793, -0.02731819, 0.99958557, 0.25917252,
            -0.99988789, 0.01215157, -0.00874858, 0.01440452,
            -0.01190754, -0.99955293, -0.02742544, 0.07365575,
            0, 0, 0, 1]
    #陀螺仪至云台转轴中心平移向量 R_g = R_i - T_iw
    T_iw: [0.04, -0.01, -0.13]
   

